modules/api/src/main/java/com/intuit/wasabi/api/ApiModule.java

```diff
@@ -101,6 +87,26 @@ protected void configure() {
 //            e.printStackTrace();
 //        }
 
+    }
+    
+    protected void installModules() {
+        LOGGER.debug("installing module: {}", ApiModule.class.getSimpleName());
+
```

Comment:
We have multiple ApiModule classes - so perhaps we can use the fully qualified name here.